lib/components/util/types.ts

```diff
@@ -95,10 +95,3 @@ export type SetViewedRouteHandler = (route?: ViewedRouteState) => void
 export type SetViewedStopHandler = (payload: { stopId: string } | null) => void
 
 export type SetLocationHandler = (payload: MapLocationActionArg) => void
-
-export interface ConfiguredTransitMode {
```

Comment:
Why is this interface being removed?

Reply:
I moved it to `config-types.ts`, and it is now called `TransitModeConfig`.